### PR TITLE
fixes 17092: Add documentation to Consul plugin integration with Rudder

### PR DESCRIPTION
**Pull request by ikramBej**
https://issues.rudder.io/issues/17092

### CODE REVIEW

**Comment by amousset** [comment]
You should not include the generated html file in the PR

File: consul/README.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* Share, synchronize and store key-value between instances.
+
+* Provide an ACL system and enable traffic encryption.
+
+* Provide a healthcheck service that return the health state of differents services.
+
+* Provides a cluster of server nodes to ensure high availability.
+
+This plugins offers the possibility to fetch key-value either from Consul server nodes or Consul agent nodes and then use them as Rudder Variables according to 
+your needs. The Rudder server does need to access the Consul server to get key-value, only the Rudder agent will do so.
```
Comment (amousset) [inline consul/README.adoc:25]: does not need

File: consul/README.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+image::docs/images/CreateKV1.jpg[]
+
+image::docs/images/CreateKV2.jpg[]
+
+image::docs/images/CreateKV3.jpg[]
+
+* *Fetch key-value via Rudder Server technique editor*:
+ 
+Now that you have finished setting up the key-value in Consul, you need to define the generic method on the rudder technique editor by configuring a variable prefix,
```
Comment (amousset) [inline consul/README.adoc:66]: Rudder

Comment (ikramBej) [reply consul/README.adoc:66]: okay fixing this

**Pull request merged**